lib/include/jxl/encode.h

```diff
@@ -70,6 +70,51 @@ typedef enum {
 
 } JxlEncoderStatus;
 
+/**
+ * Id of options to set to JxlEncoderOptions.
+ */
+typedef enum {
```

Comment:
Please add a comment here explaining (for future developers) what the options should be numbered like since it seems that you are proposing some sparse numbering for a reason.

Reply:
Removed the numeric gaps for now.

Will insert more options in upcoming MRs, numbers can still change until next release